packages/sdk-core/src/wormhole.ts

```diff
@@ -417,17 +417,30 @@ export class Wormhole extends MultiProvider<Domain> {
    *  See {@link ChainConfig.finalityThreshold | finalityThreshold} on {@link MAINNET_CONFIG | the config}
    *
    * @param msg The MessageIdentifier used to fetch the VAA
-   * @returns The ParsedVAA if available
+   * @returns The VAA bytes if available
    */
-  async getVAA(msg: MessageIdentifier): Promise<ParsedVaa | undefined> {
+  async getVAABytes(msg: MessageIdentifier): Promise<SignedVaa | undefined> {
     const { emitterChain, emitterAddress, sequence } = msg;
     const url = `${this.conf.api}/api/v1/vaas/${emitterChain}/${emitterAddress}/${sequence}`;
     const response = await axios.get(url);
 
     if (!response.data.data) return;
 
     const data = response.data.data;
-    const vaaBytes = Buffer.from(data.vaa, 'base64');
+    return Buffer.from(data.vaa, 'base64');
+  }
+
+  /**
+   * Gets a VAA from the API or Guardian RPC and parses it, finality must be met before the VAA will be available.
+   *  See {@link ChainConfig.finalityThreshold | finalityThreshold} on {@link MAINNET_CONFIG | the config}
+   *
+   * @param msg The MessageIdentifier used to fetch the VAA
+   * @returns The Parsed VAA if available
+   */
+  async getVAA(msg: MessageIdentifier): Promise<ParsedVaa | undefined> {
```

Comment:
How about calling this `getParsedVAA`? And the one above just `getVAA` (but `getVAABytes` is ok too).